web/frontend/angular/package.json

```diff
@@ -18,17 +18,18 @@
     "@angular/platform-browser": "~12.2.0",
     "@angular/platform-browser-dynamic": "~12.2.0",
     "@angular/router": "~12.2.0",
-    "@popperjs/core": "^2.11.2",
+    "@ng-bootstrap/ng-bootstrap": "^12.0.2",
```

Comment:
Worth noting that ng-bootstrap with Angular 12 ends up using Bootstrap 4. Getting Bootstrap 5 would require upgrading to Angular 13, which isn't too difficult to do but probably out of scope and not necessary in the short term.